docs/CS/SECRET-MANAGEMENT.md

```diff
@@ -0,0 +1,45 @@
+# Client credential expiration and rotation
+
+Keycloak does not provide a built-in setting for client secret expiration. To
+enforce secret rotation and prevent expired credentials from being used for
+token issuance, we have a few options:
+
+## Client Attributes for Metadata
+
+Use custom client attributes to store a `secret_expiry` timestamp. The vendor
+application create response will also include secret_expiry, similar to token expiry.
+We can make this configurable in the CMS appsettings(30 days)
```

Comment:
From what I have heard from the field, anything less than a year is operationally possible for them. Good to have it configurable.